src/nidcpower/metadata/enums_addon.py

```diff
@@ -2,5 +2,17 @@
 # Any changes to the API should be made here. enums.py is code generated
 
 enums_override_metadata = {
+    "ApertureTimeAutoMode": {"codegen_method": "no"},
```

Comment:
please add TODO pointing to Issue for adding these

Reply:
I've filed an issue for this and added a TODO comment referencing that issue in the latest commit.